lib/mdl/rules.rb

```diff
@@ -163,9 +163,15 @@
 
 rule "MD013", "Line length" do
   tags :line_length
-  params :line_length => 80
+  params :line_length => 80, :code_blocks => true
```

Comment:
Named it `code_blocks` so that it's not to be confused with lines that contain inline code elements.